app/services/admins.js

```diff
@@ -0,0 +1,24 @@
+const models = require('../models/index');
+const errors = require('../errors');
+const generatePassword = require('../helpers/users/generatePassword');
+const logger = require('../logger');
+const constants = require('../../lib/constants');
+
+module.exports.signUp = body =>
+  models.user.findOne({ where: { email: body.email } }).then(user => {
+    if (user) {
+      return user.update({ type: constants.user_types.ADMIN });
+    }
+    return models.user
+      .create({
+        firstName: body.first_name,
+        lastName: body.last_name,
```

Comment:
Not gonna ask you to do this now, but just so you know: bigger projects usually have files named mappers for converting the input params to camelCase. In this case it is not required since the params that are in snake case are few.